index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Nunito:200,300,regular,500,600,700,800,900,200italic,300italic,italic,500italic,600italic,700italic,800italic,900italic"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <title>Awesome Books</title>
+  </head>
+  <body>
+    <header>
+      <h2 class="logo">Awesome Books</h2>
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li><a class="nav-link" data-target="list">List</a></li>
+          <li><a class="nav-link" data-target="addNew">Add New</a></li>
+          <li><a class="nav-link" data-target="contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <span class="time" id="time"></span>
+    <div class="transparent">
+      <div class="main">
+        <section class="section display" id="list">
+          <h1 class="heading">All Awesome Books</h1>
+          <div id="container"></div>
+        </section>
+        <section class="section" id="addNew">
+          <h2 class="heading">Add a new book</h2>
+          <form id="form" class="form">
+            <label for="title"></label>
+            <input type="text" id="title" placeholder="Title" />
+            <label for="author"></label> <br />
+            <input type="text" id="author" placeholder="Author" /> <br />
```

Comment:
- [ ] [OPTIONAL] Please consider adding the `required` attribute to the input fields in your HTML file, this will prevent the user from adding a book without an author or the other way around.